Tavisca.Bootcamp.LanguageBasics.Exercise3/NutritionInformation.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+
+public class NutritionInformation
+{
+    public int[] Protein;
+    public int[] Carbs;
+    public int[] Fat;
+    public int[] Calories;
+   
+
+    public NutritionInformation(int[] protein, int[] carbs, int[] fat)
+    {
+        Protein = protein;
+        Carbs = carbs;
+        Fat = fat;
+        Calories = GetCalories();
+    }
+    public int[] GetCalories()
+    {
+        int[] calories = new int[Protein.Length];
+        for (int i = 0; i < Protein.Length; i++)
+            calories[i] = Carbs[i] * 5 + Protein[i] * 5 + Fat[i] * 9;
+        return calories;
+    }
+    public int GetSuitableDietIndexForPerson(string diet)
+    { 
+        int max = 0, min = 0;
+        if (diet.Length == 0)
+            return 0;
+        List<int> listone = new List<int>();
```

Comment:
What is the use of the lists?
Please read https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines for reference.